pkg/controller/traffic/traffic_shifting_status.go

```diff
@@ -0,0 +1,95 @@
+package traffic
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/labels"
+)
+
+// TODO(jgreff): calculate achieved weight based on pods in the endpoint
+// TODO(jgreff): maybe we don't need the pods themselves in Ready/NotReady
+
+type TrafficShiftingStatus struct {
+	LabeledPods   []*corev1.Pod
+	UnlabeledPods []*corev1.Pod
+
+	ReadyPods    int
+	NotReadyPods int
+
+	ReleasePods int
```

Comment:
I wonder if naming should reflect that LabeledPods and ReadyPods are of different type (the attributes are public)

Reply:
I agree. These got changed in later PRs though, si hopefully that addresses your concerns better than here